app/frontend/src/internationalization/trans/chefs/en/en.json

```diff
@@ -199,7 +199,10 @@
     "eventSubscription": "Event Subscription",
     "validEndpointRequired": "Please enter a valid endpoint starting with https://",
     "validBearerTokenRequired": "Enter a valid bearer token example: 89abddfb-2cff-4fda-83e6-13221f0c3d4f",
-    "wideFormLayout": "Wide Form Layout"
+    "wideFormLayout": "Wide Form Layout",
+    "formMetadataTitle": "Form Metadata",
+    "formMetadataMessage": "Define identifiers and labels for this form. Calls to external systems will include this metadata in their payloads.",
```

Comment:
Might this wording cause confusion with the "Labels" in the "Form Profile" section below? And what exactly is an "identifier"? Maybe it could be something like "Define JSON data to be included in calls to external systems"? JSON is maybe a little too technical, but the user doesn't know it needs to be JSON until after they enter non-JSON data and get an error. Maybe these are questions for the UX/UI folks, but I think it would be ideal if the user could figure out whether this is important to them or not (probably not) without having to read the techdocs.

Reply:
changed the wording and added i18n. yes, i think the UX team should be reviewing the text but it shouldn't be a roadblock. that stuff can get changed at any time.